index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="container">
+            <div class="logoholder">
+                <a class="mylogo" href="#top">My Logo</a>
+            </div>
+            <nav class="menuholder">
+                <img src="menuicon.png" alt="test">
+    
+            </nav>
+        </div>
+    </header>
+    
+    <section class="container2">
+        <div class="headerholder"> 
+            <h1 class="head">I’m Mariane<br>Glad to see<br> you!</h1>
+        </div>
+        <p class="headtext">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+        <div class="supporttext">
+            <b class="suptext">
+                LET'S CONNECT
+            </b>
```

Comment:
The button is a link `` <a> `` or `` <button> `` tag, and follow the button styling: Active, Hover and Pressed.